src/less/details/ds6/details.less

```diff
@@ -0,0 +1,44 @@
+@import "../../less/ds6/variables.less";
+
+details.details:not([open]) {
+    .details__summary--showless {
+        display: none;
+    }
+}
+
+details.details[open],
+details.details[data-makeup-open] {
+    .expand-btn__icon {
+        transform: rotate(180deg);
+    }
+
+    .details__summary--showmore {
+        display: none;
+    }
+
+    summary.details__summary::before {
+        content: none;
+    }
+}
+
+summary.details__summary {
+    color: @ds6-color-b4;
+
+    span.expand-btn__icon {
+        background-image: url('data:image/svg+xml;base64,@{ds6-icon-chevron-down-blue-base64}');
+    }
+
+    &::before {
+        content: none;
```

Comment:
Is this to override something set by the polyfill? Or to override browser's native styling/content?

Reply:
yeah I added that to remove IE's native styling